flink-connectors/flink-sql-connector-hbase/src/main/resources/META-INF/NOTICE

```diff
@@ -0,0 +1,27 @@
+flink-sql-connector-hbase
+Copyright 2014-2020 The Apache Software Foundation
+
+This product includes software developed at
+The Apache Software Foundation (http://www.apache.org/).
+
+This project bundles the following dependencies under the Apache Software License 2.0. (http://www.apache.org/licenses/LICENSE-2.0.txt)
+
+- commons-codec:commons-codec:1.10
+- commons-configuration:commons-configuration:1.7
+- commons-lang:commons-lang:2.6
+- commons-logging:commons-logging:1.1.3
+- com.google.guava:guava:12.0.1
+- com.yammer.metrics:metrics-core:2.2.0
+- io.netty:netty-all:4.1.44.Final
+- org.apache.hbase:hbase-common:1.4.3
+- org.apache.hbase:hbase-protocol:1.4.3
+- org.apache.hbase:hbase-procedure:1.4.3
+- org.apache.hbase:hbase-client:1.4.3
+- org.apache.hbase:hbase-prefix-tree:1.4.3
+- org.apache.htrace:htrace-core:3.1.0-incubating
```

Comment:
htrace-core bundles several dependencies but doesn't have a proper NOTICE. Maybe we should list the dependencies:

```
htrace-core
From: 'FasterXML' (http://fasterxml.com/)
  - Jackson-annotations (http://wiki.fasterxml.com/JacksonHome) com.fasterxml.jackson.core:jackson-annotations:bundle:2.4.0
    License: The Apache Software License, Version 2.0  (http://www.apache.org/licenses/LICENSE-2.0.txt)
  - Jackson-core (http://wiki.fasterxml.com/JacksonHome) com.fasterxml.jackson.core:jackson-core:bundle:2.4.0
    License: The Apache Software License, Version 2.0  (http://www.apache.org/licenses/LICENSE-2.0.txt)
  - jackson-databind (http://wiki.fasterxml.com/JacksonHome) com.fasterxml.jackson.core:jackson-databind:bundle:2.4.0
    License: The Apache Software License, Version 2.0  (http://www.apache.org/licenses/LICENSE-2.0.txt)

From: 'The Apache Software Foundation' (http://www.apache.org/)
  - Commons Logging (http://commons.apache.org/logging) commons-logging:commons-logging:jar:1.1.1
    License: The Apache Software License, Version 2.0  (http://www.apache.org/licenses/LICENSE-2.0.txt)
```

The `commons-logging` is a different version.

Reply:
I agree we should list all bundled dependencies by htrace-core, and also need to add different version of `commons-logging`